live/tests/unit/components/result-item-test.js

```diff
@@ -0,0 +1,78 @@
+import {expect} from 'chai';
+import {describe, it} from 'mocha';
+import {setupTest} from 'ember-mocha';
+
+const undefinedAnswer = 'undefined';
+const answerWithEmptyResult = {
+  value: '',
+  result: '',
+  name : 'answerWithEmptyResult',
+};
+const answerWithUndefinedResult = {
+  value: '',
+  result: undefined,
+  name : 'answerWithUndefinedResult',
+};
+const answerWithNullResult = {
+  value: '',
+  result: null,
+  name : 'answerWithNullResult',
+};
+const answerWithOkResult = {
+  value: '',
+  result: 'ok'
+};
+const answerWithRandomResult = {
+  value: '',
+  result: 'RANDOM_RESULT'
+};
+
+describe('Unit | Component | result-item-component', function () {
+  setupTest('component:result-item', {});
+
+  let component;
+
+  function initComponent() {
+    component = this.subject();
+  }
+
+  describe('#resultItemContent Computed property - undefined case', function () {
+    [
+      undefinedAnswer,
+      answerWithEmptyResult,
+      answerWithUndefinedResult,
+      answerWithNullResult
+    ].forEach(function (answer) {
+      it(`should returns false when answer provided is: ${answer.name}`, function () {
+        // given
+        initComponent.call(this);
```

Comment:
j'ai l'impression qu'il n'y a aucun intérêt à définir une méthode `initComponent` dans ce cas là. Je vois même des inconvénients : 
- ça rajoute des lignes de code ---> "moins y a de code, mieux on se porte"
- ça oblige d'utiliser `call(this)` ---> bad smell

WDYT ?